readme.markdown

```diff
@@ -178,12 +178,12 @@ git clone https://github.com/puleos/object-hash
 If you want to stand this up in a docker container, you should take at look
 at the [![node-object-hash](https://github.com/bean5/node-object-hash)](https://github.com/bean5/node-object-hash) project.
 
-### gulp tasks
+### Package scripts
```

Comment:
I'll be moving more of these scripts over to the package, rather than running everything through Gulp. Once this work is done it will simplify the setup a little, so there is no dependency on Gulp any longer.

Reply:
👍   

It would be nice to remove the dependency on Gulp.